### PR TITLE
Create yunlaige.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/yunlaige.com.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="yunlaige.com">
+
```
Comment (pipboy96) [inline src/chrome/content/rules/yunlaige.com.xml]: No `\n` here.

File: src/chrome/content/rules/yunlaige.com.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+	<target host="yunlaige.com" />
+	<target host="www.yunlaige.com" />
+	<target host="m.yunlaige.com" />
+
```
Comment (pipboy96) [inline src/chrome/content/rules/yunlaige.com.xml]: If possible:
`	<securecookie host=".+" name=".+" />`

File: src/chrome/content/rules/yunlaige.com.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<ruleset name="yunlaige.com">
+
+	<target host="yunlaige.com" />
```
Comment (pipboy96) [inline src/chrome/content/rules/yunlaige.com.xml]: Remove the empty line before this line.

**Comment by Bisaloo** [comment]
Thank you!

**Pull request merged**